### PR TITLE
Added PowerToys stuff

### PR DESCRIPTION
**Pull request by julsehn**

### CODE REVIEW

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
This isn't the relevant article to add these tips in. I'd move it to the "Useful Windows Tips" page under one section related to PowerToys as a whole (not an exhaustive documentation of its functionality), and reword 3/4 of the changes here.

**Comment by AvogatoWizardWhisker** [comment]
We could make a dedicated page to recommend useful apps that improves productivity. Making a list for PowerToys specifically in `useful-windows-tips` is a little bit too much.

**Comment by itsWindows11** [comment]
It's outside of this PR's scope but definitely doable.

**Comment by AvogatoWizardWhisker** [comment]
I can see that but why is it committed to `keyboard-shortcut.md`?

**Comment by julsehn** [comment]
> I can see that but why is it committed to `keyboard-shortcut.md`?

These tools are helpful for having in hand some shortcuts and create others.

**Comment by aervnu** [comment]
> > I can see that but why is it committed to `keyboard-shortcut.md`?
> 
> These tools are helpful for having in hand some shortcuts and create others.

I mean that's true but there are no fixed keyboard shortcuts in PowerToys since they're all user defined, and the Keyboard Manager itself *should* be included in another page specifically about recommended apps including PT like Avogato suggested.

File: src/wiki/keyboard-shortcuts.md [hunk 1]
```diff
@@ -206,3 +206,18 @@ This article will introduce you to the essential shortcuts that will help you na
 | <kbd>Win</kbd> + <kbd>Ctrl</kbd> + <kbd>→</kbd>  | Switches between virtual desktops to the right. |
 | <kbd>Win</kbd> + <kbd>Ctrl</kbd> + <kbd>←</kbd>  | Switches between virtual desktops to the left.  |
 | <kbd>Win</kbd> + <kbd>Ctrl</kbd> + <kbd>F4</kbd> | Closes the current virtual desktop.             |
+
+## How to find them easily with PowerToys
```
Comment (AvogatoWizardWhisker) [inline src/wiki/keyboard-shortcuts.md:210]: This has

File: src/wiki/keyboard-shortcuts.md [hunk 2]
```diff
@@ -206,3 +206,18 @@ This article will introduce you to the essential shortcuts that will help you na
...
+PowerToys is a great tool that every Windows should have, and it’s developed by Microsoft, that has very powerful tools that will make you more productive.
```
Comment (AvogatoWizardWhisker) [inline src/wiki/keyboard-shortcuts.md:211]: Tbf, the header "How to find them easily with PowerToys" isn't relevant to the wiki page. You will not find every shortcuts mentioned in PowerToys.

Comment (julsehn) [reply src/wiki/keyboard-shortcuts.md:211]: It only will show the commands with win key

Comment (AvogatoWizardWhisker) [reply src/wiki/keyboard-shortcuts.md:211]: This is getting really complicated 😵‍💫

[the remark this answers is not in the document]
Comment (sthivaios) [reply src/wiki/keyboard-shortcuts.md:211]: How about "a very powerful set of tools" instead of "has very powerful tools"?

**Pull request closed without merging**